platform/kube/conversion.go

```diff
@@ -35,11 +35,11 @@ const (
 
 	// KubeServiceAccountsOnVMAnnotation is to specify the K8s service accounts that are allowed to run
 	// this service on the VMs
-	KubeServiceAccountsOnVMAnnotation = "alpha.istio.io/serviceaccounts/kubernetes"
+	KubeServiceAccountsOnVMAnnotation = "alpha.istio.io/kubernetes-serviceaccounts"
```

Comment:
should be serviceaccount ? no s ? (there is only 1 value)

Reply:
Actually there can be multiple service accounts on VM to run the same service, which is the same as in K8s :)

Reply:
what is the syntax for multiple accounts and what does it mean to have multiple accounts ?

Reply:
For example, you can have one VM running by foo@gserviceaccount.com, and one VM running by bar@gserviceaccount.com. Both VMs are the endpoints for service "datastore". In this case, the service needs to specify both service accounts in the service's annotation.